docs/examples.rst

```diff
@@ -119,11 +123,18 @@ In this example we add the headers to allow the Application to impersonate a use
         :param dict configuration: a configuration dictionary
         :param str user_uuid: a User UUID per the platform
         """
-        full_url = "https://mcc.example.org/v1/service"
+        full_url = "https://mcc.example.org/service"
+        headers = {
+            "Accept": "application/json",
+            "Mcc-Version": "v2019-03-22",
+            "MCC-Impersonate": "com:mdsol:users:{}".format(user_uuid)
```

Comment:
we blacklist this header! Do not want them to impersonate

Reply:
Why not?  Having apps perform activities on behalf of a user seems to be one of the primary goals of initiatives like API-gateway

Reply:
Updated in https://github.com/mdsol/requests-mauth/pull/14/commits/b1da514750fd6ae9a19a6f7fd64eae446b995e03 to keep the impersonation example as is for non-API Gateway services. The API Gateway Lambda is already doing impersonation on behalf of the requester